src/test/compile-fail/privacy1.rs

```diff
@@ -14,6 +14,9 @@
 #[lang="sized"]
 pub trait Sized {}
 
+#[lang="copy"]
+pub trait Copy {}
```

Comment:
Is there no way the compiler could pretend to have `Sized` or `Copy`?
Not that I wouldn't use libcore for this, but sometimes libcore doesn't compile and I need a testcase.